pyrogram/client/types/messages_and_media/message.py

```diff
@@ -3108,3 +3111,45 @@ def pin(self, disable_notification: bool = None) -> "Message":
             message_id=self.message_id,
             disable_notification=disable_notification
         )
+
+    def get_poll_option_votes(self, option: PollOption):
+        """Get votes for attached public poll option
```

Comment:
Maybe "Get votes for an option attached to a public poll"? A bit easier to comprehend